weekThree/api_client.py

```diff
@@ -0,0 +1,14 @@
+import requests
+
+class APIClient:
+    def __init__(self, url):
+        self.url = url
+
+    def fetch_data(self):
+        try:
+            response = requests.get(self.url)
+            response.raise_for_status()
+            return response.text
+        except requests.RequestException as e:
```

Comment:
Рaving general RequestException catch block is not perfect, because there can be network errors, server errors, client errors, not found (404, which some times is not considered as an error). It is better to have atleast a few more specialized exception handlers.